tools/make/image.mk

```diff
@@ -67,15 +67,17 @@ image.push.%: image.build.%
 
 .PHONY: image.multiarch.verify
 image.multiarch.verify:
-	$(eval pass := $(shell ))
-	docker buildx --help | grep -qw "docker buildx" || { \
-		echo "Cannot find `docker buildx`, please install first"; \
-		exit 1; }
+	$(eval PASS := $(shell docker buildx --help | grep "docker buildx" ))
+	@if [ -z "$(PASS)" ]; then \
+		echo "Cannot find docker buildx, please install first"; \
+		exit 1;\
+	fi
 
 .PHONY: image.multiarch.emulate $(EMULATE_TARGETS)
 image.multiarch.emulate: $(EMULATE_TARGETS)
 $(EMULATE_TARGETS): image.multiarch.emulate.%:
-	docker run --rm --privileged tonistiigi/binfmt --install linux/$* # Install QEMU emulator, the same emulator as the host will report an error but can safe ignore
+# Install QEMU emulator, the same emulator as the host will report an error but can safe ignore
+	docker run --rm --privileged tonistiigi/binfmt --install linux/$* 
```

Comment:
@Xunzhuo can you remove the extra whitespace here.

Reply:
line 80? I do not think this is extra, command needs it in makefile.